apps/fabric-website/src/pages/Styles/Fabric7Page/docs/web/Fabric7Overview.md

```diff
@@ -1,19 +1,19 @@
-### What’s new in Fabric 7?
+### What’s new in Fluent UI 7?
 
-We’ve updated Fabric to include a new set of Web Fluent design styles, which include colors, type, depth and motion which are applied to the standard set of Fabric controls. These include:
+We’ve updated Fluent UI to include a new set of Web Fluent design styles, which include colors, type, depth and motion which are applied to the standard set of Fluent UI controls. These include:
 
 - Colors are an updated, shared set used in common components such as personas, sites, groups and calendar events. We’ve also updated the Neutrals to be warmer and more approachable.
 - Typography has been updated to a new common type ramp used across core Microsoft apps, leveraging the latest type weights from Fluent.
 - Depth styles now include a common set of shadow levels from Fluent, and are applied to context menus, dialogs, panels, etc.
 - Motion styles have been updated to the latest Fluent styles and are available to be used inside of your applications.
 
-### Upgrading to Fabric 7
+### Upgrading to Fluent UI 7
 
-Along with a visual refresh, Fabric 7 updates the minimum required version of React to 16.8.1 and TypeScript to 3.3. It also includes a number of behavior improvements and removes old deprecated props. For a full rundown of these changes, and upgrade instructions, see the [release notes](https://github.com/microsoft/fluentui/wiki/Fabric-7) on our wiki.
+Along with a visual refresh, Fluent UI 7 updates the minimum required version of React to 16.8.1 and TypeScript to 3.3. It also includes a number of behavior improvements and removes old deprecated props. For a full rundown of these changes, and upgrade instructions, see the [release notes](https://github.com/microsoft/fluentui/wiki/Fabric-7) on our wiki.
 
-### Can I preview the Fluent styles using Fabric 6?
+### Can I preview the Fluent styles using Fluent UI 6?
 
-If you want to preview the Fluent styles and how they will affect your existing Fabric 6 site, follow the [Fluent customization steps](https://github.com/microsoft/fluentui/wiki/Fluent-theme-in-Fabric-6) outlined in our wiki.
+If you want to preview the Fluent styles and how they will affect your existing Fluent UI 6 site, follow the [Fluent customization steps](https://github.com/microsoft/fluentui/wiki/Fluent-theme-in-Fabric-6) outlined in our wiki.
```

Comment:
```suggestion
If you want to preview the Fluent styles and how they will affect your existing Fabric 6 site, follow the [Fluent customization steps](https://github.com/microsoft/fluentui/wiki/Fluent-theme-in-Fabric-6) outlined in our wiki.
```

Reply:
confirmed with David, 5x and 6x will be refered to as Fabric